projects/documentation/content/_includes/partials/accessibility.njk

```diff
@@ -0,0 +1,84 @@
+            </sp-table-cell>
+            <sp-table-cell>
+                    <sp-button static="white" style="--mod-bold-font-weight:400">Tests that ensure additional states of the component are accessible. 
+                    This could be interactive states of a component or its multiple variants.</sp-button>
+            </sp-table-cell>
+            <sp-table-cell>
+            <sp-button   id="trigger-2" variant="secondary" style="--mod-button-background-color-default: #a7f0ba;">Tested</sp-button>
+            <sp-overlay trigger="trigger-2@hover" placement="bottom">
+                <sp-tooltip>
+                    Passes all automated tests with no reported accessibility violations.
+                </sp-tooltip>
+            </sp-overlay>
+            </sp-table-cell>
+        </sp-table-row>
+         <sp-table-row value="row2">
+            <sp-table-cell></sp-table-cell>
+            <sp-table-cell>
+                <sp-button static="white" style="--mod-bold-font-weight:400">Keyboard navigation</sp-button>
+            </sp-table-cell>
+            <sp-table-cell>
+                    <sp-button static="white" style="--mod-bold-font-weight:400">Tests that ensure focus is properly managed, and all interactive functions of a 
+                        component have a proper keyboard-accessible equivalent.</sp-button>
+            </sp-table-cell>
+            <sp-table-cell>
+             <sp-button id="trigger-3" variant="secondary" style="--mod-button-background-color-default: #a7f0ba;">Tested</sp-button>
+            <sp-overlay trigger="trigger-3@hover" placement="bottom">
+                <sp-tooltip>
+                    Passes all automated tests with no reported accessibility violations.
+                </sp-tooltip>
+            </sp-overlay>
+            </sp-table-cell>
+        </sp-table-row>
+        <sp-table-row value="row3">
+            <sp-table-cell></sp-table-cell>
+            <sp-table-cell>
+                <sp-button static="white" style="--mod-bold-font-weight:400">Screen reader</sp-button>
+            </sp-table-cell>
+            <sp-table-cell>
+                    <sp-button static="white" style="--mod-bold-font-weight:400">This manual testing ensures that the visual information on the screen 
+                    is properly conveyed and read correctly by screen readers such as JAWS, VoiceOver, and NVDA.</sp-button>
+            </sp-table-cell>
+            <sp-table-cell>
+            <sp-button id="trigger-4" variant="secondary" style="--mod-button-background-color-default: #9ef0f0;">Manually Tested</sp-button>
+            <sp-overlay trigger="trigger-4@hover" placement="bottom">
+                <sp-tooltip>
+                    A human has manually tested this component, e.g. screen reader testing.
+                </sp-tooltip>
+            </sp-overlay>
+            </sp-table-cell>
+        </sp-table-row>
+    </sp-table-body>
+</sp-table>
```

Comment:
```suggestion
    <sp-table role="table" quiet style="--mod-table-row-background-color-hover: none">
    <sp-table-head>
        <sp-table-head-cell>Component</sp-table-head-cell>
        <sp-table-head-cell>Accessibility test</sp-table-head-cell>
        <sp-table-head-cell>Accessibility Contract</sp-table-head-cell>
        <sp-table-head-cell>Status</sp-table-head-cell>
    </sp-table-head>
    <sp-table-body>
        <sp-table-row value="row1">
            <sp-table-cell role="cell"><sp-link href="https://opensource.adobe.com/spectrum-web-components/components/{{componentName}}/#usage">{{componentName}}</sp-link></sp-table-cell>
            <sp-table-cell role="rowheader">
               Default state
            </sp-table-cell>
            <sp-table-cell role="cell">
                Test(s) that ensure the initial render state of a component is accessible.
                </sp-table-cell>
            <sp-table-cell role="cell">
                <sp-button id="trigger-1" variant="secondary" style="--mod-button-background-color-default: #a7f0ba;">Tested</sp-button>
                <sp-overlay trigger="trigger-1@hover" placement="bottom">
                    <sp-tooltip>
                        Passes all automated tests with no reported accessibility violations.
                    </sp-tooltip>
                </sp-overlay>
            </sp-table-cell>
        </sp-table-row>
        <sp-table-row value="row2">
            <sp-table-cell role="cell"></sp-table-cell>
            <sp-table-cell role="rowheader">
                Advanced states
            </sp-table-cell>
            <sp-table-cell role="cell">
                    Tests that ensure additional states of the component are accessible. 
                    This could be interactive states of a component or its multiple variants.
            </sp-table-cell>
            <sp-table-cell role="cell">
            <sp-button id="trigger-2" variant="secondary" style="--mod-button-background-color-default: #a7f0ba;">Tested</sp-button>
            <sp-overlay trigger="trigger-2@hover" placement="bottom">
                <sp-tooltip>
                    Passes all automated tests with no reported accessibility violations.
                </sp-tooltip>
            </sp-overlay>
            </sp-table-cell>
        </sp-table-row>
         <sp-table-row value="row2">
            <sp-table-cell role="cell"></sp-table-cell>
            <sp-table-cell role="rowheader">
                Keyboard navigation
            </sp-table-cell>
            <sp-table-cell role="cell">
                    Tests that ensure focus is properly managed, and all interactive functions of a 
                        component have a proper keyboard-accessible equivalent.
            </sp-table-cell>
            <sp-table-cell role="cell">
             <sp-button id="trigger-3" variant="secondary" style="--mod-button-background-color-default: #a7f0ba;">Tested</sp-button>
            <sp-overlay trigger="trigger-3@hover" placement="bottom">
                <sp-tooltip>
                    Passes all automated tests with no reported accessibility violations.
                </sp-tooltip>
            </sp-overlay>
            </sp-table-cell>
        </sp-table-row>
        <sp-table-row value="row3">
            <sp-table-cell role="cell"></sp-table-cell>
            <sp-table-cell role="rowheader">
                Screen reader
            </sp-table-cell>
            <sp-table-cell role="cell">
                    This manual testing ensures that the visual information on the screen 
                    is properly conveyed and read correctly by screen readers such as JAWS, VoiceOver, and NVDA.
            </sp-table-cell>
            <sp-table-cell role="cell">
            <sp-button id="trigger-4" variant="secondary" style="--mod-button-background-color-default: #9ef0f0;">Manually Tested</sp-button>
            <sp-overlay trigger="trigger-4@hover" placement="bottom">
                <sp-tooltip>
                    A human has manually tested this component, e.g. screen reader testing.
                </sp-tooltip>
            </sp-overlay>
            </sp-table-cell>
        </sp-table-row>
    </sp-table-body>
</sp-table>
```